CMakeLists.txt

```diff
@@ -303,7 +303,7 @@ add_link_options(LINKER:-z,text)
 
 if("${SERENITY_ARCH}" STREQUAL "i686")
     add_compile_options(-march=i686)
-else()
+elseif("${SERENITY_ARCH}" STREQUAL "x86_64")
```

Comment:
do we need to add a new `-march` option for aarch64 here (...later)?

Reply:
I haven't seen `-march=` used in arm64 builds. I think `-march=` generally tells gcc "you can assume the following arch as lowest supported ISA version", so "i686" means gcc can pick instructions that don't exist in i386, i486, i586 chips. For 32-bit arm builds, you can pick armv7-a or armv6. For aarch64, there's only one version so far as far as I know, so it isn't needed yet.

(For x86_64, we also pass the minimum 64-bit target which is also a no-op. So I think we don't really need this for x86_64 builds either atm (?))